presentation/backend/src/presentation-backend/domain/RulesetElements.ts

```diff
@@ -20,15 +22,19 @@ export class Ruleset extends DefinitionElement {
 
   /**
    * Generates a unique code for a ruleset
-   * @param modelId     - ID of a the model this ruleset should be created in
-   * @param rulesetId   - ID of the ruleset code is being created for (to ensure uniqueness for different rules)
-   * @param iModelDb    - db the ruleset is supposed to be inserted into
+   * @param iModelDb DB the ruleset is supposed to be inserted into
+   * @param modelId ID of a the model this ruleset should be created in
+   * @param ruleset The ruleset code is being created for
    */
-  public static createRulesetCode(modelId: Id64String, rulesetId: string, iModelDb: IModelDb) {
+  public static createRulesetCode(iModelDb: IModelDb, modelId: Id64String, ruleset: PresentationRuleset) {
+    let codeValue = ruleset.id;
+    if (ruleset.version)
+      codeValue += `.${normalizeVersion(ruleset.version)}`;

```

Comment:
I think `codeValue` would be easier to scan quickly with the eyes if we used a different separator than what's already present in the version string. How about `@`?

Reply:
Agree. Changed to `@`.

Reply:
Does this require a change in native side?

Reply:
No, CodeValue is only used to ensure uniqueness. It's not used used in native code.